Doppler.Sap/appsettings.json

```diff
@@ -35,7 +36,9 @@
           "Endpoint": "BusinessPartners"
         },
         "BillingConfig": {
-          "Endpoint": "Invoices"
+          "Endpoint": "Invoices",
+          "NeedCreateIncomingPayments": "true",
```

Comment:
You are using a string for a boolean value.

```suggestion
          "NeedCreateIncomingPayments": true,
```

Reply:
yes, my bad. I will update it.

Reply:
Done